src/backend/app.js

```diff
@@ -30,24 +30,24 @@ if (process.env.API_PATH) {
   throw "API_PATH is not set. Remember to set it in your .env file"
 }
 app.get("/future-meals", async (req, res) => {
-  const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` > NOW()");
-  rows[0].length === 0 ? res.sendStatus(404) : res.send(rows.slice(0, 1))
+  const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` >= NOW()");
+  rows[0].length === 0 ? res.status(200).send(`No meals found with future dates`) : res.send(rows.slice(0, 1))
```

Comment:
For suppose, while data fetching if something went wrong, there will be an error and we should handle that error to convey it to the user.
To handle errors: we can use `try/catch` or `if/else`.
Think about it and let me know if you have any questions.

Reply:
I got it,we can apply try catch here,then do we need for all queries individually or one for all? Please advice,thanks.